### PR TITLE
Add PHP sample and reorganize sections

### PR DESCRIPTION
**Pull request by pchico83**

### CODE REVIEW

**Review by rlamana — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,25 +2,32 @@
 
 This repository contains examples of how to develop applications directly in your Kubernetes cluster using [okteto](https://github.com/okteto/okteto), with different platforms and deployment technologies.
```
Comment (rberrelleza) [inline README.md]:
```suggestion
This repository contains examples of how to develop applications directly in your Kubernetes cluster using [okteto](https://github.com/okteto/okteto), with different programming languages, frameworks and deployment tool.
```

File: README.md [hunk 2]
```diff
@@ -2,25 +2,32 @@
...
 
-## Scenarios
+## By languages
```
Comment (rberrelleza) [inline README.md]:
```suggestion
## Programming Languages
```

File: README.md [hunk 3]
```diff
@@ -2,25 +2,32 @@
...
+1. [Golang](https://github.com/okteto/go-getting-started)
 1. [Java Gradle](https://github.com/okteto/java-gradle-getting-started)
 1. [Java Maven](https://github.com/okteto/java-maven-getting-started)
-1. [Node](https://github.com/okteto/node-getting-started/blob/master/README.md)
-1. [PHP + Webpack](php/README.md)
+1. [Node](https://github.com/okteto/node-getting-started)
+1. [PHP](https://github.com/okteto/php-getting-started)
 1. [Python](python/README.md)
+
+## By frameworks
```
Comment (rberrelleza) [inline README.md]:
```suggestion
## Frameworks
```

File: README.md [hunk 4]
```diff
@@ -2,25 +2,32 @@
...
+
+1. [Django](django/README.md)
+1. [Multiple services with Node and React](https://github.com/okteto/movies)
+1. [PHP + Webpack](php/README.md)
 1. [Ruby on Rails](ruby/README.md)
 
-## Advanced scenarios
+## By deployment tools
```
Comment (rberrelleza) [inline README.md]:
```suggestion
## Deployment Tools
```

File: README.md [hunk 5]
```diff
@@ -2,25 +2,32 @@
...
 
-1. [Multiple services with Node and React](https://github.com/okteto/movies/blob/master/README.md)
 1. [Helm](helm/README.md)
+1. [OpenFaaS](https://medium.com/okteto/how-to-develop-a-serverless-app-with-openfaas-and-okteto-d85435f0eca1)
+
+## Remote
```
Comment (rberrelleza) [inline README.md]:
```suggestion
## Remote IDEs
```

File: README.md [hunk 6]
```diff
@@ -2,25 +2,32 @@
...
+1. [Multiple services with Node and React](https://github.com/okteto/movies)
+1. [PHP + Webpack](php/README.md)
 1. [Ruby on Rails](ruby/README.md)
 
-## Advanced scenarios
+## By deployment tools
 
-1. [Multiple services with Node and React](https://github.com/okteto/movies/blob/master/README.md)
 1. [Helm](helm/README.md)
+1. [OpenFaaS](https://medium.com/okteto/how-to-develop-a-serverless-app-with-openfaas-and-okteto-d85435f0eca1)
```
Comment (rberrelleza) [inline README.md:28]: I'd put this on a 'Serverless' section. While not wrong, I don't think most people relate functions with a deployment tool but rather as a development model.

**Review by rberrelleza — CHANGES_REQUESTED** [review]
Not sure what problem we are trying to solve here, but if you like this order better go for it 👍

**Pull request merged**